src/AlgebraicGeometry/Schemes/ProjectiveSchemes/Objects/Attributes.jl

```diff
@@ -326,8 +289,8 @@ end
     
 Return a `CoveredScheme` ``X`` isomorphic to `P` with standard affine charts given by dehomogenization. 
 
-Use `dehomogenize(P, U)` with `U` one of the `affine_charts` of ``X`` to 
-obtain the dehomogenization map from the `graded_coordinate_ring` of `P` 
+Use [`dehomogenization_map(::ProjectiveScheme, ::AbsSpec)`](@ref) with `U` one of the `affine_charts` of ``X`` to
```

Comment:
Maybe this will fix the doctest error?
```suggestion
Use [`dehomogenization_map(::AbsProjectiveScheme, ::AbsSpec)`](@ref) with `U` one of the `affine_charts` of ``X`` to
```

Reply:
Unfortunately not, and neither 5 other things I tried. Since this wasting too much of my time, I disable the reference.